roles/common/meta/main.yml

```diff
@@ -10,10 +10,13 @@ dependencies:
       - repo: 'deb {{ apt_repos.sensu.repo }} sensu main'
         key_url: '{{ apt_repos.sensu.key_url }}'
         validate_certs: no
+    when: ansible_architecture != "ppc64le"
   - role: docker
     when: docker.enabled is defined and docker.enabled|bool
+    when: ansible_architecture != "ppc64le"
```

Comment:
For ppc64 should just disable docker in env configs, rather than arch conditionalize.